src/cmp_ccpa.js

```diff
@@ -166,27 +225,39 @@ const checkPage = async (pageType, url) => {
 	await clearCookies(page);
 
 	// Now we can run our tests.
-	log(`Test 1 start: Adverts load and the CMP is displayed on initial load`);
+	log(`[TEST 1] start: CMP loads and the ads are NOT displayed on initial load`);
```

Comment:
Same in the US/CCPA region the ads should load even before the CMP has been interacted with.